<launch>
    <node pkg="boustrophedon_server" name="boustrophedon_server" type="boustrophedon_planner_server" output="screen" >
        <param name="outline_layer_count" value="1"/>
        <param name="stripe_separation" value="1.0"/>
        <param name="stripe_angle" value="0.0"/>
    </node>
</launch>
